src/includes/java/performance/getting-started/java.mdx

```diff
@@ -166,3 +166,63 @@ try {
     span.finish()
 }
 ```
+
+### Connecting Errors with Spans
+
+Sentry errors can be linked with transactions and spans.
+
+Errors reported to Sentry while transaction or span is running are linked automatically:
+
+```java {tabTitle: Java}
+ISpan span = Sentry.startTransaction("task");
+try {
+    processItem();
+} catch (Exception e) {
+    Sentry.captureException(e);
+} finally {
+    span.finish();
+}
+```
+
+```kotlin {tabTitle: Kotlin}
+val span = Sentry.startTransaction("task")
+try {
+    processItem()
+} catch (e: Exception) {
+    Sentry.captureException(e)
+} finally {
+    span.finish()
+}
+```
+
+In future, if a thrown exception should be reported to Sentry after the span has already finished, you must link the exception with the span by calling `setThrowable` method to connect the error:
```

Comment:
What do you mean by `in future` here? 
Is it related to the fact we rethrow outside of the span?

Reply:
I am also not convinced about this paragraph, it's been discussed at: https://github.com/getsentry/sentry-docs/pull/2818#discussion_r555243258

Originally I wrote it as:

> In cases where the decision if thrown exception should be reported to Sentry happens later in the future after span is already finished, to connect error with a span you must link the exception with a Span by calling `setThrowable` method:

Reply:
tbh, both sentences are not ideal, I still don't get the meaning of it (before/after) the change, at first I thought it was only about me not fully understanding how the Spans work but now I do a bit more, and when I read this phrase, I still don't get it.

Reply:
would you mind rephrasing @maciejwalkowiak so I can understand & help what we are trying to explain?

Reply:
Exceptions may be thrown within spans that can finish before exception gets reported to Sentry. To attach span information to this exception, you must link it by calling `setThrowable` method.

Reply:
> Exceptions may be thrown within spans that can finish before exception gets reported to Sentry. To attach span information to this exception, you must link it by calling `setThrowable` method.

I'd change to this last one, way easier to read and understand, thanks